pkgs/build-support/setup-hooks/make-binary-wrapper/default.nix

```diff
@@ -10,7 +10,7 @@
 
 makeSetupHook {
   name = "make-binary-wrapper-hook";
-  propagatedBuildInputs = [ dieHook ];
+  propagatedBuildInputs = [ dieHook cc ];
```

Comment:
Is this to make it work in stdenvNoCC?

Reply:
@artemst can you comment? here's the original commit https://github.com/rhelmot/nixpkgs/commit/4e0769e941b81817c0372cf833d73476ea92e756

Reply:
@artemist can you comment? here's the original commit https://github.com/rhelmot/nixpkgs/commit/4e0769e941b81817c0372cf833d73476ea92e756

Reply:
Yes, the tests and several normal packages use makeBinaryWrapper in runCommand. This was a simpler change than making all of them runCommandCC